applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -815,10 +789,10 @@ static void on_all_states(struct modem_msg_data *msg)
 	if (IS_EVENT(msg, cloud, CLOUD_EVT_USER_ASSOCIATED)) {
 
 		/* Re-enable low power features after cloud has been associated. */
-		int err = configure_low_power();
+		int err = lte_lc_psm_req(true);
```

Comment:
Should it be possible to disable PSM by setting `CONFIG_LTE_PSM_REQ=n`? If yes, then this would still enable PSM after user association, so you could check the Kconfig option here.

Reply:
Good catch, fixed